libs/api/domains/driving-license-book/src/models/activeBookId.response.ts

```diff
@@ -0,0 +1,12 @@
+import { Field, ObjectType } from '@nestjs/graphql'
+@ObjectType()
+export class BookId {
```

Comment:
This goes for other GraphQL fields as well ... the name of an object type (field) should rather be Book, instead of BookId ... then you fetch the id of the book with book.id